test/data/MonotoneMuseum_Korean.toml

```diff
@@ -249,7 +249,7 @@ rands = [
 [[ test ]]
 game_system = "MonotoneMuseum:Korean"
 input = "DTS"
-output = "歪み表(海)(56) ＞ 【酒の海】\n海の水が酒になる。水を飲んだすべての者は酔っ払い理性を失う。その様はまるで暗黒期に滅亡した“怠惰の古都”を彷彿とさせる。"
+output = "일그러짐표(바다)(56) ＞ 【술의 바다】\n바닷물이 술이 된다. 물을 마신 모든 것은 취해 이성을 잃는다. 그 모습이 마치 암흑기에 멸망한 "나태의 고도"를 연상시킨다."
```

Comment:
The `"`s inside a string must be escaped, or please use `“...”` (fullwidth quotation marks) as in Japanese output.